pdk/1.3/modules/ROOT/pages/policies-pdk-configure-features-timer.adoc

```diff
@@ -6,7 +6,7 @@ endif::[]
 
 Use the `Clock` injectable to build a timer to enable delayed, periodic, and synchronous functions. Each policy supports only one timer.
 
-Each timer provides the following two functions:
+Each timer provides the following two async functions:
```

Comment:
Following what we are adding in line 62, we should use the wording "methods" instead of "functions".